portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -13,6 +13,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hello world! Fetch() Worked!");
```

Comment:
After re-reading the instructions for this part, I think you're to create a new servlet that maps to a URL and essentially duplicate this logic. You'll be returning structured JSON data from this new servlet in the next step.